iron-doc-viewer.html

```diff
@@ -119,6 +119,22 @@
         observer: '_descriptorChanged',
       },
 
+      /**
+       * Set to true when running in the catalog.
+       *
+       * By default, hash link includes the component name,
+       * so links work when an `iron-component-page` is
+       * served statically. In the catalog, the component name
+       * is omitted because it's already included in a URL parameter.
+       *
+       * In the catalog, the app handles routing &
+       * scrolling to the selected anchor.
+       */
+      catalog: {
```

Comment:
How about just having an `anchorPrefix` property, and `iron-component-page` can data bind it to a computed binding like:

    prefix="[[computePrefix(descriptor.is,catalog)]]"

Where `computePrefix` returns either the empty string or something like `Polymer.Base--`

Why would we want an anchor with no hash?